### ISSUE TITLE
Film VFI breaking loops

### ISSUE THREAD

**Issue by LuisBurdallo** [Fannovel16/ComfyUI-Frame-Interpolation#44]
I was wondering if it could be possible to have the interpolation to avoid breaking a perfect loop. I guess by not creating frames after the final frame? Don't know how it works internally but it seems to break the perfect loops I create with AnimateDiff.

**Comment by smbell1979** [comment]
I just came here to see if there was any info on this myself!  I'd love to see any option that interpolated between the first and last frames as well. I think this might solve the issue.

**Comment by LuisBurdallo** [comment]
> I just came here to see if there was any info on this myself! I'd love to see any option that interpolated between the first and last frames as well. I think this might solve the issue.

Yes interpolating from last and first would be a good option if possible.

**Comment by saruzaru** [comment]
would it be possible to add a Loop toggle to the VFI nodes? (or at least the FILM VFI)
i'm generating loops and then interpolating but there are a few missing frames at the end
all it would need internally is adding the first frame after the last frame, generate the missing frames and deleting the last frame so that there's no doubling of the first frame.

**Comment by ak3dde** [comment]
![FilmVFI_fix](https://github.com/Fannovel16/ComfyUI-Frame-Interpolation/assets/149612368/1a181e74-a27b-4f7c-a313-60bccea0edab)

Found a solution: By duplicating the first frame of the perfect loop and adding it to the end of the sequence, FilmVFI is keeping the perfect seemless loop in place. That would be a nice addtion to the nodes.

**Comment by marcosclips** [comment]
> ![FilmVFI_fix](https://private-user-images.githubusercontent.com/149612368/335844165-1a181e74-a27b-4f7c-a313-60bccea0edab.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU3ODk2NTgsIm5iZiI6MTcyNTc4OTM1OCwicGF0aCI6Ii8xNDk2MTIzNjgvMzM1ODQ0MTY1LTFhMTgxZTc0LWEyN2ItNGY3Yy1hMzEzLTYwYmNjZWEwZWRhYi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwOFQwOTU1NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jOGUxYmY5ZjQ5OTgzY2QzMWZlNWRiOThkMzk2MzFmOGM5NjNiNGRhMGEyZjkzZTA3OTVhODliNWJiZWNmOWFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.KlKng_ICtZoQFZwYJxXHJexLUpFFUDq58xfSONAE_lc)
> 
> Found a solution: By duplicating the first frame of the perfect loop and adding it to the end of the sequence, FilmVFI is keeping the perfect seemless loop in place. That would be a nice addtion to the nodes.

Hello, that's a nice feature I see in 2 other Interpolators, but not in Film. The problem is that the other interpolators don't work with Windows, so I was trying your approach.
I don't seem to get it to work though.

How exactly do I add the first frame to the end of the sequence?
And do I do so before, or after the Interpolation?
